input/pagecontent/construction_des_flux.md

```diff
@@ -1,7 +1,7 @@
 ### Flux 1
 
-Description du flux avec schémas et liens hypertextes vers le  <a href="st_flux1.html">flux 01</a>
+Description du flux avec schémas et liens hypertextes vers le <a href="st_flux1.html">flux 01</a>
```

Comment:
En Markdown, on évite dans la mesure du possible d'ajouter des balises HTML.

Un lien s'écrit `[flux 01](./st_flux1.html)`, mais je ne sais pas ça fonctionne dans l'IG ?

Reply:
Oui ça fonctionne effectivement! :)
Le mieux est d'utiliser un maximum md mais c'est parfois limitant, par exemple pour faire des tableaux un peu sympa

J'ai supprimé cette page car il s'agit d'un relicat de l'IG modele

Reply:
(Mais pas top pour la maintenance, les tableaux en html...)

Reply:
Yes. Il faudrait tester comment est généré en HTML un tableau en Markdown, et après jouer sur le fichier CSS.

Reply:
Très bonne idée!!
La principale limite, c'est pour fusionner plusieurs cases ensemble, on ne peut pas le faire en markdown.. On pourrait possiblement jouer avec le CSS mais ça me paraît compliqué

Reply:
> Yes. Il faudrait tester comment est généré en HTML un tableau en Markdown, et après jouer sur le fichier CSS.
j'ai testé et c'était pas concluant pour des tableaux complexes :(

Reply:
pour tester https://tableconvert.com/fr/markdown-to-html

Reply:
J'ai supprimé le lien pour éviter à avoir à le gérer et risquer les erreurs (les liens n'étaient plus valides)